Lib/os.py

```diff
@@ -28,6 +28,8 @@
 
 from _collections_abc import _check_methods
 
+GenericAlias = type(list[int])
```

Comment:
Why this module, and e.g. `_collections_abc` use this code but other use `import types; type.GenericAlias`?

Reply:
The "right" way is to import it from `types`, but we can't use that here. The `os` module is part of a small set of modules that are imported at startup. The `types` module is not part of that set, and there's a unit test that checks this. (In test/test_site.py, in class `StartupImportTests`, look at ``test_startup_imports`.)